### PR TITLE
Add Thor 1.3.1

### PR DESCRIPTION
**Pull request by vladimyr**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/thor.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  auto_updates true
+
+  app 'Thor.app'
+
+  postflight do
+    system_command 'xattr',
+                   args: [
+                           '-dr', 'com.apple.quarantine',
+                           "#{appdir}/Thor.app"
+                         ]
```
Comment (vladimyr) [inline Casks/thor.rb:19]: Unless I remove quarantine attribute every time I try to run it I get _Thor.app is damaged and can’t be opened. You should move it to the Trash_ message 🤷

Comment (vitorgalvao) [reply Casks/thor.rb:19]: Don’t remove the quarantine. We’re the ones adding it, so it makes no sense. It’s one more for https://github.com/Homebrew/homebrew-cask/issues/70798.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/thor.rb (hunk 1 above)
Comment (vladimyr) [reply Casks/thor.rb:19]: Ok, but that makes cask unusable, should we mark it with _do not merge_ label or should I close it for now?

Comment (vitorgalvao) [reply Casks/thor.rb]: > but that makes cask unusable

Not if installed with the `--no-quarantine` flag.

Comment (vladimyr) [reply Casks/thor.rb]: True that 👍

**Pull request merged**